Rasa_Bot/README.md

```diff
@@ -12,18 +12,23 @@ The steps are as follows:
    - The output for the above command should be something like this: [{"recipient_id":"user","text":"Hoi Kees!"},{"recipient_id":"user","text":"Sure, you should ...
    - See [this page](https://rasa.com/docs/rasa/connectors/your-own-website#restinput) for details on how to use the REST channel.
 
+Note that while the requirements-file lists Rasa 2.0.2 as a requirement, this is only needed to train a language model.
```

Comment:
You also need rasa to serve the agent right? Or to develop with it?

Reply:
@svenvanderburg You need Rasa for training the language model. For developing, Rasa is handy if you plan to do something bigger. For example, if you want to create a new Rasa agent from scratch it is useful if you can directly create an empty project structure for the agent via the command line based on Rasa. What do you mean with "serving the agent"?

Reply:
With "serving the agent" I mean running the rasa server and action server, i.e. running the application.
If you are editing rasa code you also want to test or debug things by running the application or running tests. You do not always want to rely on docker for that, often you just want to use your own python environment.